2020-InfiniteRecharge/src/main/java/frc/robot/Robot.java

```diff
@@ -186,9 +186,12 @@ public void teleopPeriodic()
       powerCell.feeder();
   }
       //Compressor
-     if(compressorEnable){
+     if(compressorEnable && compressorControl.useCompressor){
       compressorControl.enable();
     }
+    else if(compressorEnable){
+      compressorControl.disable();
+    }
```

Comment:
Can the `compressorControl.useCompressor` check be in an `if/else` inside the `if (compressorEnable)`?

Reply:
That is now done